mypy.ini

```diff
@@ -41,7 +41,7 @@ files =
 #
 # `exclude` is a regex, not a list of paths like `files` (sigh)
 #
-exclude = torch/include/|torch/csrc/|torch/distributed/elastic/agent/server/api.py
+exclude = torch/include/|torch/csrc/|torch/distributed/elastic/agent/server/api.py|torch/testing/_internal
```

Comment:
Just curious - what's the point of removing torch/testing/_internal from mypy checks?

Reply:
I'll create a post